libs/application/templates/marriage-conditions/src/dataProviders/NationalRegistryMaritalStatusProvider.ts

```diff
@@ -0,0 +1,110 @@
+import { MaritalStatus, NationalRegistryPerson } from '../types/schema'
+import {
+  BasicDataProvider,
+  SuccessfulDataProviderResult,
+  FailedDataProviderResult,
+  Application,
+} from '@island.is/application/types'
+import { getValueViaPath } from '@island.is/application/core'
+import { MarriageConditionsFakeData, YES } from '../types'
+
+export class NationalRegistryMaritalStatusProvider extends BasicDataProvider {
```

Comment:
Our thought process was keeping this here for now, and then once we start moving `dataProviders` to `template-api-modules `that a `nationalRegistryXroadUser` provider should be added. That would contain the marital status so that everyone can use it. I am a bit concerned that all applications are using the national registry soap client to get info about applicants.